js/package.json

```diff
@@ -106,14 +106,15 @@
     "p-queue": "^6.6.2",
     "p-retry": "4",
     "semver": "^7.6.3",
-    "uuid": "^10.0.0"
+    "uuid": "^10.0.0",
+    "langchain": "^0.3.3",
```

Comment:
Synced with Jacob to double check. Steps to handle this:

1. Rm from dependencies
2. Make the llm evaluator a separate entrypoint (see traceablea nd the create_entrypoints.js script)
3. We should probably refrain from importing init chat model from the top since that requires langchain to be installed as well as the sub modules. Maybe accept a chat model as an argument and have them initialize, and in our docs we show using that. Core I think more OK